src/django_components/util/html.py

```diff
@@ -1,100 +0,0 @@
-from typing import List, Union
```

Comment:
These functions can be removed, as their equivalent was implemented in selectolax. So I removed also the test file.